src/fileformats/ocd_file_format.cpp

```diff
@@ -45,6 +72,16 @@ OcdFileFormat::OcdFileFormat()
 	// Nothing
 }
 
+OcdFileFormat::OcdFileFormat(quint16 version)
+: FileFormat { MapFile, idForVersion(version),
+               version == 1 ? ::OpenOrienteering::ImportExport::tr("OCAD legacy export")
```

Comment:
Legacy export should mention the version number. E.g. "OCAD legacy exporter (v8)". Not every user knows...

Reply:
Proposal: "OCAD version 8 (old exporter)". 

Anyway, we should remove the legacy exporter as soon as possible. On Android, we don't have this dialog at all.